homeassistant/components/philips_js/device_trigger.py

```diff
@@ -30,7 +32,7 @@
 
 async def async_get_triggers(
     hass: HomeAssistant, device_id: str
-) -> list[dict[str, Any]]:
+) -> GetAutomationsResult:
```

Comment:
I would like to revert this, for reasoning and the review see: <https://github.com/home-assistant/core/pull/72163#issuecomment-1133865649>

Reply:
I have reworked this to remove the incorrect GetAutomation*** aliases